server/controllers/offlinePayment.controllers.js

```diff
@@ -39,6 +42,36 @@ const OfflinePaymentController = {
       })
       .catch((error) => next(error));
   },
+
+  resolveOfflinePayment(req, res, next) {
+    const offlinePaymentId = req.params.id;
+    const adminId = req.user._id;
+    resolveOfflinePayment({ offlinePaymentId, adminId })
+      .then((payment) => {
+        res.status(httpStatus.OK).json({ success: true, message: 'Payment resolved', payment });
+      })
+      .catch((error) => next(error));
+  },
+
+  makeComment(req, res, next) {
+    const comment = req.locals;
+    const { user } = req;
+    makeComment({ comment, user })
```

Comment:
Since you used comment raised, I feel `raiseComment` is a better method name 🤷🏽‍♂️ 
```suggestion
    raiseComment({ comment, user })
```